tiny_dnn/core/framework/tensor.h

```diff
@@ -41,6 +41,14 @@ class Tensor {
   Tensor() {}
 
   /**
+   * Constructor that accepts a storage object and create a Tensor housing
+   * that storage. Commonly used by [] operator.
+   * @param storage object, can be xtarray.
+   * @return
+   */
+  explicit Tensor(Storage s) : storage_(s) {}
```

Comment:
The scope of the argument variable is pretty limited. So I don't think it is a must to use full-length word here.

A lot of people have written books about programming styles, argued on internet forums. Maybe it's like pigeons talking about proper manners for their droppings.

But, hey, why the argument isn't a referenced type? We are scheming how to produce heat with computer processors and copying tensor is a hefty operation.